tests/tests/test_utils.py

```diff
@@ -33,3 +36,40 @@ def test_parent_template_from_variable(self):
                 'patterns/base.html',
             ],
         )
+
+    @override_settings(TEMPLATES=[
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'APP_DIRS': True,
+        },
+    ])
+    def test_get_template_dirs_app_dirs(self):
+        template_dirs = ['/'.join(d.replace(os.path.dirname(settings.BASE_DIR), 'dpl').split('/')[-4:-1]) for d in get_template_dirs()]
```

Comment:
This is a bit opaque. Could we add a comment describing what this line is doing?

Reply:
Indeed 🤦  This does two things:

- Absolute path -> relative path
- Rewrite the current project’s paths so they are normalised regardless of the root folder’s name (dependent on how the git repository is cloned)